test/regress/regress1/sygus-abduct-test.smt2

```diff
@@ -1,4 +1,4 @@
-; COMMAND-LINE: --sygus-abduct --sygus-abort-size=2 
+; COMMAND-LINE: --produce-abducts --sygus-stream --sygus-abort-size=2
```

Comment:
I am wondering, from a usability standpoint, whether it makes sense to "mix" abduction options with SyGuS options. 
Isn't it a bit weird that the user asks for an abduct with `(get-abduct ...)` but then to get a stream of abducts, a SyGuS options needs to be enabled? 
SyGuS here is the technique, and the user asking for an abduct shouldn't really care about it.

Maybe there is room for an additional command `(get-abducts ...)` (that will automatically turn on the required sygus flag, if that's a possibility)? 
Or `(get-abducts ... N)` where  N is `sygus-abort-size`?

Reply:
We haven't standardized the "streaming" version of get-abduct.  In fact, it probably does not make sense from an API point of view to tell CVC4 to do an infinite loop streaming solutions.

What you said would require a further discussion to the smt2 interface, but I don't think it's necessary to make this public facing.